mono_repo/lib/src/mono_config.dart

```diff
@@ -132,9 +137,18 @@ class MonoConfig {
       );
     }
 
+    final ci = {
+      if (json.containsKey('travis')) CI.travis,
+      if (json.containsKey('github')) CI.github,
+    };
+
     final travis = json['travis'] ?? {};
 
-    if (travis is! Map) {
+    if (travis is bool) {
```

Comment:
Consider breaking this out into a function, the check is identical between the two CI configs (you would have to pass in the `ci` set so you can remove them, or add a `!= false` check above).

Reply:
ack